src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/SystemUnicastIPAddressInformation.cs

```diff
@@ -43,7 +43,7 @@ internal SystemUnicastIPAddressInformation(Interop.IpHlpApi.IpAdapterUnicastAddr
 
         public override IPAddress Address { get { return _innerInfo.Address; } }
 
-        public override IPAddress IPv4Mask
+        public override IPAddress? IPv4Mask
```

Comment:
Can this really return `null`? In the ctor, `if (ipAddress.AddressFamily == AddressFamily.InterNetwork)`, then `_ipv4Mask` is set to non-null. Then below, the AddressFamily is checked, and if it isn't `InterNetwork`, then `Any` is returned, else `_ipv4Mask` which is set above. So I'm not sure how `null` could be returned.

Reply:
Right, good catch thanks